profiles/schema.py

```diff
@@ -385,7 +392,7 @@ def mutate_and_get_payload(cls, root, info, **input):
 class CreateProfileMutation(relay.ClientIDMutation):
     class Input:
         service_type = graphene.Argument(AllowedServiceType, required=True)
-        profile = ProfileInput(require=True)
+        profile = ProfileInput(requirde=True)
```

Comment:
Sorry but I laughed a bit to this fix :D

Reply:
Hesus christos 🤣 😆 🤣